### ISSUE TITLE
How to add another PHP version not working.

### ISSUE THREAD

**Issue by subashcpoudel** [leokhoa/laragon#348]
**Issue**:
When going to Menu -> PHP -> Version [.....] -> How to add another PHP version, it sends to a non existing link  ( https://github.com/leokhoa/laragon//topic/166 ) on the browser.

**Laragon Version**: 
Laragon Full 6.0 220916

**Comment by Lewis8379** [comment]
I was about to report the same issue. There are other dead links in the software too.

For example, Menu > MySQL > PhpMyAdmin opens the browser to https://github.com/leokhoa/laragon//topic/98 

I tried removing the double slash but it is still invalid.

**Comment by Lewis8379** [comment]
The same dead links also show on their website

https://laragon.org/docs/operations.html

**Comment by reiniermybeats** [comment]
I think the forum, which these links pointed to before, is now redirecting to github for some reason.

**Comment by Pen-y-Fan** [comment]
The forum has been down for a week 😔 I don't know why.

The easiest way for a beginner to set up Laragon see [How to Set Up THE BEST Windows 11 Development Environment for PHP 8+ with Laragon & Scoop](https://www.youtube.com/watch?v=n2hmnTMZiwg) or a blog: [How to Set Up The Best PHP 8+ Development Environment in Windows 11](https://webartisan.info/how-to-set-up-the-best-php-8-development-environment-in-windows-11). It includes using a section on downloading different PHP versions, by using a package manager (Scoop). 

This isn't the way I download PHP, I would download from [windows PHP.net download](https://windows.php.net/download/), unzip the file to C:\laragon\php, **stop** Laragon, switch PHP versions and **start all**. However, you need to know which version of PHP to download, which runtime (VS16), architecture (e.g. x64) and thread safety (TS/NTS), see the left-hand panel on the download site for a detailed explanation, the short (TLDR) version:

1. Visual Studio C++ run time - PHP source code is compiled using Microsoft Visual Studio, the current version of Visual
   Studio 2019 complies to **VC16**.
2. Architecture - Laragon Full version is **64-bit (x64)**, the portable version is **32-bit**.
3. Thread safety:
    - Apache with mod_php requires a Thread Safe (TS) build.
    - NGINX, IIS and Apache with mod_fastcgi requires a Non-Thread-Safe (NTS) build.

So if you want PHP 8.0 64-bit for Apache, download **VS16 x64 Thread Safe (2022-Oct-25 11:29:02)**  **Zip** \[25.63MB\]

@leokhoa please can you give an update on the problem with the forum?

**Comment by leokhoa** [comment]
@Pen-y-Fan : I was trying to upgrade the forum to v2 of NodeBB but failed. The current one is too old so it may have security issues.

**Comment by reiniermybeats** [comment]
@leokhoa Thank you for your hard work. I for one will patiently wait until all is fixed. Let us know if we can help with anything!

**Comment by aditbaco** [comment]
yeah the forum is down for quite sometime now... kindly help to fix it @leokhoa

**Comment by Pen-y-Fan** [comment]
@aditbaco the method to upgrade Laragon I use is:

1. backup MySQL (Heidi > Connect > Right click your connection > Export Database as SQL > Select all the databases you want to export (except **mysql** and **sys**) > Tick Database(s): Create, Table(s): Create, Data: Insert, Output: ZIP compressed .SQL file, Filename: give it a name and location, and Export.
2. uninstall Laragon (Control Panel > uninstall a program > Laragon > Uninstall)
3. reinstall Laragon 6 in the same location [How to set up Laragon on a new Windows computer](https://pen-y-fan.github.io/2022/11/07/how-to-set-up-laragon-on-a-new-windows-computer/)
4. If the databases don't upgrade to MySQL 8 import them from the backup.

**Comment by flik** [comment]
Download php version from here as per Your OS 64bit or 32bit then try. [https://windows.php.net/downloads/releases/)](https://windows.php.net/downloads/releases/)

It is working perfectly for me.

**Comment by alxndr-w** [comment]
Up-to-date information is provided here:
https://github.com/leokhoa/laragon/wiki

This issue could be closed by the original author.
